posthog/test/test_feature_flags.py

```diff
@@ -2334,6 +2337,97 @@ def test_capture_is_called(self, patch_decide, patch_capture):
             disable_geoip=None,
         )
 
+    
+    @mock.patch.object(Client, "capture")
+    @mock.patch("posthog.client.decide")
+    def test_capture_is_called_in_get_feature_flag_payload(self, patch_decide, patch_capture):
```

Comment:
full suite of tests confirming that `get_feature_flag_payload` sends events with actual data in the responses.